frontend/src/App.vue

```diff
@@ -70,7 +54,7 @@
           <div class="diagonal">
           </div>
           <div class="bottom-buttons-col" style="display:flex;align-items: center;">
-            <v-btn id="back" :disabled="promptIndex<1 || isReplaying" @click="back" v-show="false && !shouldShowGeneratorSelection()">
+            <v-btn id="back" :disabled="promptIndex<1 || isReplaying" @click="back" v-show="false">
```

Comment:
there should be a flag for not showing 'back' btn at 1st page.

Reply:
Done